modules/developer_manual/pages/app/advanced/l10n.adoc

```diff
@@ -5,7 +5,7 @@ https://www.transifex.com/projects/p/owncloud/[Transifex]. To start
 translating sign up and enter a group. If translations for your community app should be
 added to Transifex follow the steps decribed at the end of this page.
 
-[[php]]
+[[translation-php]]
```

Comment:
Is there a reason why you named them ``translation`` instead of ``l10n`` while the others are more or less named after the filename like ``storage-backend``.
Means as an example ``translation-php`` (btw underline instead of dash) --> ``l10n_php``

Reply:
I think there was at the time, but I don't remember them atm.